builtin/bisect--helper.c

```diff
@@ -22,11 +22,11 @@ static GIT_PATH_FUNC(git_path_bisect_names, "BISECT_NAMES")
 
 static const char * const git_bisect_helper_usage[] = {
 	N_("git bisect--helper --bisect-reset [<commit>]"),
-  N_("git bisect--helper --bisect-next-check <good_term> <bad_term> [<term>]"),
+ 	N_("git bisect--helper --bisect-next-check <good_term> <bad_term> [<term>]"),
```

Comment:
This change (and the others in "Update bisect--helper.c") looks like it wants to be squashed into a previous patch rather than living in its own commit.

Reply:
Ah, I see, you resolved the merge conflicts via GitHub's interface. In regular projects using GitHub for a PullRequest-reliant workflow, that would be okay, but in Git, we actually cannot use those conveniences. We have to rebase the patches.

Reply:
Sorry!! I will fix this up